test/ci/test_tiktok.py

```diff
@@ -0,0 +1,17 @@
+import unittest
+import youtube_dl
+
+
+class TikTokTestYoutubeDl(unittest.TestCase):
+    def test_meta_data(self):
+        url = 'https://www.tiktok.com/@danieltbraun/video/6817099671043853574'
+        params = {}
+        ydl = youtube_dl.YoutubeDL(params)
+        info = ydl.extract_info(url, download=False)
+        self.assertEquals(info['share_count'], 121)
```

Comment:
Please extend test to check all fields.

Reply:
suggest changing the assert to greater-than so that another share will not break it